main.py

```diff
@@ -45,6 +45,7 @@
 SkillBot.shuffle(1)
 SkillBot.wiggle(1)
 SkillBot.detect_color()
+SkillBot.richards_uber()
```

Comment:
I think that we should remove this from main.py because this could confuse people if they don't know to press a button. I have an idea that I will show in another pull request.
```suggestion
```